go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	github.com/coreos/etcd v3.3.13+incompatible
 	github.com/coreos/go-oidc v2.2.1+incompatible
 	github.com/coreos/pkg v0.0.0-20180928190104-399ea9e2e55f
-	github.com/dexidp/dex v0.0.0-20201118094123-6ca0cbc85759
+	github.com/dexidp/dex v0.0.0-20210628213014-ceb4324c183d
```

Comment:
where is this version from? I don't see it on https://openbase.com/go/github.com/dexidp/dex/@/versions

Reply:
hmm, I thought that SHA was tagged at v2.29.0 but it turns out it's one of the parents of that merge commit - I'll fix this to be the v2.29.0 commit instead.